cmake/test/ngagetest.c

```diff
@@ -0,0 +1,33 @@
+// The purpose of this test is to validate that there are no undefined references
+// when building with the N-Gage SDK since SDL can only be linked statically.
+
+#define IMPORT_C
+#include "SDL.h"
```

Comment:
This should be `<SDL3/SDL.h>`, should it not?

Reply:
Yep, thanks.